cypress.config.ts

```diff
@@ -9,5 +9,9 @@ export default defineConfig({
     defaultCommandTimeout: 60000,
     viewportWidth: 1200,
     baseUrl: "http://localhost:3000",
+    watchForFileChanges: false,
+    env: {
```

Comment:
It would be better if we passed variables from `.env` file. We can do this via [dotenv](https://www.npmjs.com/package/dotenv) package. Please ping me in DM, and I'll help you with this one